classes/patreon_user_profiles.php

```diff
@@ -76,18 +74,11 @@ function save_patreon_user_profile_fields( $user_id ) {
 		// update_user_meta( $user_id, 'user_lastname', $_POST['postalcode'] );
 	}
 
-
-
 	function prevent_email_change( $errors, $update, $user ) {
 
-		$old = get_user_by('id', $user->ID);
+		$old = get_user_by( 'id', $user->ID );
 
-		if( $user->user_email != $old->user_email   && (!current_user_can('create_users')) )
+		if( $user->user_email != $old->user_email   && ( ! current_user_can( 'create_users' ) ) )
```

Comment:
Nope, it doesnt. It follows the earlier style guideline of spacing the ! operator. But as i mentioned, we can ditch this one if it would be better to read.